### ISSUE TITLE
depends

### ISSUE THREAD

**Issue by hayden-t** [primaeval/skin.naked#1]
Hi, i tried installing this on krypton, and i get message depends xbmc.gui 5.10.0 Very keen to try this, as just what i need for the aged care place i work.

**Comment by hayden-t** [comment]
tried just changing to .12 and it installed but would not load. musnt be compatible, its fine, dont worry about it

**Comment by primaeval** [comment]
I haven't touched it for a couple of years. I think it was for Jarvis on Android.
What are you trying to run it on?

**Comment by hayden-t** [comment]
Yeah I could see that from last commit times. Android. Don't worry about it, I've changed my mind about wanting to use it
